back/.gitignore

```diff
@@ -1 +1 @@
-node_modules/
+back/node_modules/
```

Comment:
Esto ya está ignorado en `back/.gitignore`

Reply:
on it